app/controllers/v0/institution_programs_controller.rb

```diff
@@ -9,7 +9,7 @@ def autocomplete
       @data = []
       if params[:term].present?
         @search_term = params[:term]&.strip&.downcase
-        @data = InstitutionProgram.version(@version[:number]).autocomplete(@search_term)
+        @data = InstitutionProgram.autocomplete(@search_term)
```

Comment:
since this is being removed should `InstitutionProgram::autocomplete` be updated to account for the version?

Reply:
Any suggestions on the best way to do so here?